pyvista/examples/downloads.py

```diff
@@ -7472,7 +7472,7 @@ def download_whole_body_ct_male(load=True):  # pragma: no cover
 
     >>> label_map = dataset['label_map']
     >>> label_map.get_data_range()
-    (0, 117)
+    (np.uint8(0), np.uint8(117))
```

Comment:
I guess the issue here is that `get_data_range` returns an array with the same dtype as the array from which the range is computed.

instead of `tuple(array)` we could do `tuple(array.tolist())` and let numpy convert to the nearest built-in type. See https://numpy.org/doc/stable/reference/generated/numpy.ndarray.tolist.html.

For 1-d arrays this is fine, but for multi-dimensional arrays we need to be careful since a nested list will be generated, but `tuple` won't be recursively applied to the nested lists. For these cases `_validation.validate_array(array, to_tuple=True)` can be used,  or maybe even just `_validation._cast_array._cast_to_tuple` which is used by that function internally.

Reply:
`get_data_range` in all cases does a `return value1, value2`, so it was most straightforward to do something like `return value1.item(), value2.item()`.  We now also use `math.nan` to avoid returning the `np.nan`, which messes up the typing.

Reply:
> We now also use `math.nan` to avoid returning the `np.nan`, which messes up the typing.

Can you please elaborate on this? Because
```
>>> type(np.nan)
<class 'float'>
```

Reply:
I believe you are right, but now this is moot as we are using numpy scalars here.